### ISSUE TITLE
set2key error with next release of data.table 1.13.0

### ISSUE THREAD

**Issue by mattdowle** [gdemin/expss#67]
Hi Gregory,

I'm working on data.table release to CRAN and am testing revdeps.  expss will fail as follows:

```
./expss.Rcheck/00check.log:* checking whether the package can be loaded ... ERROR
./expss.Rcheck/00check.log:Loading this package had a fatal error status code 1
./expss.Rcheck/00check.log:Loading log:
./expss.Rcheck/00check.log:Error : 'set2key' is not an exported object from 'namespace:data.table'
./expss.Rcheck/00check.log:Error: package or namespace load failed for ‘expss’:
./expss.Rcheck/00check.log: unable to load R code in package ‘expss’
./expss.Rcheck/00check.log:Execution halted
./expss.Rcheck/00check.log:* DONE
./expss.Rcheck/00check.log:Status: 1 ERROR
```

This is due to note 2 in NEWS :

> 2. `set2key`, `set2keyv`, and `key2` have been removed, as they have been warning since v1.9.8 (Nov 2016) and halting with helpful message since v1.11.0 (May 2018). When they were introduced in version 1.9.4 (Oct 2014) they were marked as 'experimental' and quickly superceded by `setindex` and `indices`.

As far as I can tell, you don't actually use `set2key` in `expss` but it fetches it from data.table namespace. Hope it's easy for you to remove `set2key`.

Thanks, Matt

**Comment by gdemin** [comment]
Hi, Matt
Thanks, I will fix this issue as soon as possible.

**Comment by gdemin** [comment]
Fixed in v0.10.6
